<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>robotsetDlg</class>
 <widget class="QDialog" name="robotsetDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>400</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>机器人IP:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>机器人端口:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>机器人远程IP:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="robot_ip">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="robot_port">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="robot_remoteip">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="robot_model">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>无机器人</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>智昌机器人</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>机器人型号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>(1477-1496)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>姿态标准:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="robot_posure_model">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>350</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>安川姿态标准</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>库卡姿态标准</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>松下姿态标准</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>robotsetDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
